mungegithub/mungers/blunderbuss.go

```diff
@@ -162,6 +162,19 @@ func selectMultipleOwners(potentialOwners weightMap, weightSum int64, count int)
 	return owners
 }
 
+func getPotentialOwnersHelper(author *string, feats *features.Features, files []*githubapi.CommitFile, issueNumber *int) (weightMap, int64) {
+	potentialOwners, weightSum := getPotentialOwners(*author, feats, files, true)
+	if len(potentialOwners) == 0 {
+		potentialOwners, weightSum = getPotentialOwners(*author, feats, files, false)
+		if len(potentialOwners) == 0 {
+			glog.Errorf("No OWNERS found for PR %d", *issueNumber)
+			return nil, 0
```

Comment:
can you just return `potentialOwners, weightSum` here? It should be equivalent anyway.